src/components/modals/ClassInfo/ClassInfo.tsx

```diff
@@ -332,6 +335,22 @@ export default function ClassInfo({
                     </Typography>
                 </Box>
             )}
+            {user === undefined && !isInThePast && (
+                <Alert severity="info" sx={{ mt: 1 }} icon={<Login fontSize={"small"} />}>
+                    Du må logge inn for å kunne booke eller legge til timer i timeplanen
+                </Alert>
+            )}
+            {user !== undefined && userConfig === undefined && !isInThePast && (
+                <Alert severity="info" sx={{ mt: 1 }}>
+                    <AlertTitle>
+                        Registrer ditt <b>{chain}</b>-medlemskap
```

Comment:
```suggestion
                        Koble til <b>{chain}</b>-medlemskap
```

Looks good, but I think we should avoid using "Registrer" to be clear that you are not creating a new 3T membership, but rather giving us access to your account 🤠

Reply:
Yeah, that could be confusing. I ended up using "koble til" in the header, and "tilknytte"/"knyte" (to rezervo) in the subtitles. Updated the above comment with new screenshots now. 

Implemented in: https://github.com/mathiazom/rezervo-web/pull/100/commits/705ba6b890a2d48123f450938b54a70929b84b6b

Reply:
Edit: ended up on using "koble til" both places See https://github.com/mathiazom/rezervo-web/pull/100/commits/8d21b24ee78138d124ecbe72a5308274dcea2698 and updated screenshots above